uncontrast_studies/models/consciousness_measure.py

```diff
@@ -21,7 +22,7 @@ class UnConsciousnessMeasureSubType(models.Model):
     type = models.ForeignKey(
         blank=False,
         null=False,
-        on_delete=CASCADE,
+        on_delete=PROTECT,
```

Comment:
why?

Reply:
because if the "type" above is deleted while it still has sub types connected to it, I want the delete to be blocked